addons/l10n_mx/migrations/2.1/post-migrate.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+from odoo.addons.account.models.chart_template import update_taxes_from_templates
+
+
+def migrate(cr, version):
+    update_taxes_from_templates(cr, 'l10n_mx.mx_coa')
```

Comment:
Why updating taxes ? This method is only for updating tax's tags, I don't see the relation with this PR ? Am I missing something ?